pubspec.yaml

```diff
@@ -1,23 +1,23 @@
 name: streamqflite
 description: reactive stream wrapper around sqflite inspired by sqlbrite
-version: 1.0.0
+version: 2.2.3
```

Comment:
Curious why 2.2.3? I guess to match `sqflite`? I don't think we need to actually track that closely with versions so `2.0.0` would be fine.